package.json

```
@@ -20,7 +20,7 @@
     "test-live": "TEST_ON_DEV_BUILD=1 npm run test",
     "unit-test": "tsc -p ./tsconfig-test.json && mocha $(find ./build/test -name '*-test.js') --require source-map-support/register || true",
     "puppeteer-test": "tsc -p ./tsconfig-test.json && mocha --timeout 30000 ./build/test/test/run.js --require source-map-support/register",
-    "lerna:publish": "npm install && npm run build && npm run test && lerna publish --no-private"
+    "release": "npm install && npm run build && npm run test && changeset publish"
```

Not strictly part of this PR but the script looks weird :)

```suggestion
    "prerelease": "npm run build && npm run test",
    "release": "changeset publish"
```

`install` is an absolute curiosity. I'd move `build` to individual package but alas